go/pull_request.go

```diff
@@ -372,3 +388,49 @@ func (h *PullRequestHandler) backportPR(ctx context.Context, event github.PullRe
 
 	return nil
 }
+
+func (h *PullRequestHandler) createDocsPreview(ctx context.Context, event github.PullRequestEvent, prInfo prInformation) error {
+	// TODO: Checks:
+	// 1. Is a PR against either:
+	// 	- vitessio/vitess:main
+	//	- vitessio/vitess:release-\d+\.\d+
+	// 2. PR contains changes to either `go/cmd/**/*.go` OR `go/flags/endtoend/*.txt`
+	return nil
+}
+
+func (h *PullRequestHandler) updateDocs(ctx context.Context, event github.PullRequestEvent, prInfo prInformation) (err error) {
+	installationID := githubapp.GetInstallationIDFromEvent(&event)
+	client, err := h.NewInstallationClient(installationID)
+	if err != nil {
+		return err
+	}
+
+	ctx, logger := githubapp.PreparePRContext(ctx, installationID, prInfo.repo, event.GetNumber())
+	defer func() {
+		if e := panicHandler(logger); e != nil {
+			err = e
+		}
+	}()
+
+	// TODO: Checks:
+	// - is vitessio/vitess:main branch OR is vitessio/vitess versioned tag (v\d+\.\d+\.\d+)
+	// - PR contains changes to either `go/cmd/**/*.go` OR `go/flags/endtoend/*.txt`
+	if prInfo.base.GetRef() != "main" {
+		logger.Debug().Msgf("PR is merged to %s, not main, skipping website cobradocs sync", prInfo.base.GetRef())
+		return nil
```

Comment:
let's add the number of the Pull Request in the debug message to ease debugging. this can be useful when you've got plenty of logs.

Reply:
good call!